common/primitives/timestamp/time.go

```diff
@@ -175,7 +175,7 @@ func TimePtr(t time.Time) *time.Time {
 
 func TimeValue(t *time.Time) time.Time {
 	if t == nil {
-		return time.Time{}
+		return time.Unix(0, 0).UTC()
```

Comment:
parts of the system use `IsZero` to determine if the time is set, like a `nil` check.